models/lorenz_attractor.nestml

```diff
@@ -0,0 +1,31 @@
+neuron lorenz_attractor:
+
+initial_values:
+  x real = 1
+  y real = 1
+  z real = 1
+end
+
+equations:
+  shape foo = exp(-t)
+  x' = sigma * (y - x) / s
+  y' = (x * (rho - z) - y) / s
+  z' = (x * y - beta * z) / s
+end
+
+update:
+  integrate_odes()
+end
+
+parameters:
+  sigma real = 10
+  beta real = 8/3
+  rho real = 28
+end
+
+input:
+end
+
+output: spike
```

Comment:
is it possible to make a model without output? would be cleaner

Reply:
@jougs what is the output of rate neurons in NEST? (my guess is that it is a continuous variable)
We might want to keep ``output`` for later but make it optional, with ``spike`` as the default output

Reply:
Sorry for being unclear (or too short). My complete idea was to
1. remove the `output` block
2. add functions `emit_rate()`, `emit_current()` and similar once we need them
3. forbid the use of more than one of them in a single model
4. infer the type of output from the `emit_` function used
5. create the corresponding code for the connection handshake
6. probably warn if none of the `emit_` functions is used.